src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs

```diff
@@ -244,15 +238,17 @@ private IServiceProvider ConfigureIOC(IServiceCollection services)
                 config.For<IReferenceDataTokenService>().Use<ReferenceDataTokenService>()
                     .Ctor<IClientConfiguration>().Is(Configuration.ReferenceDataApiAuthentication);
 
-                config.For<IRoatpTokenService>().Use<RoatpTokenService>()
-                    .Ctor<IClientConfiguration>().Is(Configuration.RoatpApiAuthentication);
+                config.For<SFA.DAS.Http.Configuration.IManagedIdentityClientConfiguration>().Use(Configuration.RoatpApiAuthentication);
```

Comment:
This won't work when we have more IManagedIdentityClientConfiguration derived classes it should config the RoatpApiClientConfiguration and then inject that into the RoatpApiClientFactory

Reply:
Ah i see, good spot. Using RoatpApiClientConfiguration now and injecting that into the RoatpApiClientFactory